### PR TITLE
scenario edits based on copyedit feedback

### PR DESCRIPTION
**Pull request by danielsyoder**
*Issue #, if available:*
N/A - typos, clarity, and formatting
*Description of changes:*
Edits based on copyedit feedback.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/scenarios/Bank Payments/references.en.md [hunk 1]
```diff
@@ -16,21 +16,26 @@ The access patterns in the scenario are outlined as:
...
 
-**[Using GSIs to Fulfill Secondary Access Patterns](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/GSI.html)**
-**[Working with Queries](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Query.html)**
-**[Write Shard a GSI for Selective Queries](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/bp-indexes-gsi-sharding.html)**
+
+#### Helpful references
+
+Given the above, see below for some helpful references.
+
+- **[Best Practices for Using Sort Keys to Organize Data](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/bp-sort-keys.html)**
```
Comment (switch180) [inline content/scenarios/Bank Payments/references.en.md:38]: @danielsyoder is it okay if I put these. back in. a bulleted list? The list - char. was removed and I added it back

File: content/scenarios/Retail Cart/_index.en.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ weight = 1
...
 
 - Users work with items that can be `ACTIVE`, `SAVED`, or `PURCHASED`. Once they are `PURCHASED` they will be assigned an *OrderId*.
-- Items have the following: *AccountID*, *Status* (`ACTIVE`/`SAVED`/`PURCHASED`), *CreateTimestamp*, & *ItemSKU* - (Total size is < 1 KB)
-- When a user logs in they view the active items in their cart – organized by most recently added.
-- Users can view items that they have saved for later – organized by most recently saved items.
-- Users can view items that they have purchased – also organized by most recently purchased items.
-- Product teams have the ability to regularly query across all users to identify the users that have a specific product either `ACTIVE`, `SAVED`, or `PURCHASED`.
-- The Business Intelligence team needs to run a number of ad hoc complex queries against the dataset to create weekly and monthly reports.
+- Items have the following: *AccountID*, *Status* (`ACTIVE`, `SAVED`, or `PURCHASED`), *CreateTimestamp*, & *ItemSKU* (Total size is < 1 KB).
```
Comment (switch180) [inline content/scenarios/Retail Cart/_index.en.md:12]: This line right here, officer. This is the one!
Seriously, though - what's our formatting standard for attribute names and. attribute values? Here you use courier. new for values and bold for attribute names. Is that a purposeful decision? If so I need to revert a change I made. to another file.

File: content/scenarios/Bank Payments/references.en.md (hunk 1 above)
Comment (danielsyoder) [reply content/scenarios/Bank Payments/references.en.md:38]: yes

**Comment by switch180** [comment]
Ship it!

Merging.

**Pull request merged**